test/account-add.test.js

```diff
@@ -24,7 +24,10 @@ test('account.add()', async (t) => {
       subdomain: 'person',
     })
 
-    assert.equal(peer.db.account.has({ account, keypair: keypair2 }), false)
+    assert.equal(
+      await p(peer.db.account.has)({ account, keypair: keypair2 }),
```

Comment:
Could all these public APIs be made `async` instead of callback-based which then requires `promisify()`?

E.g. `records()` is an async iterable, not a pull-stream or other callback-based approach.

Reply:
ah you mean them not having cb's at all? i thought we wanted cbs internally?

Reply:
i guess related too https://github.com/staltz/ppppp-db/issues/26 (but that issue was suggesting having promise+cb)

Reply:
merging, open to discussing this more later. i like promises, just want to understand the reasons you have avoided them in the past first